kubernetes/prometheus/prometheus.yaml

```diff
@@ -0,0 +1,23 @@
+global:
+  scrape_interval: 15s
+  scrape_timeout: 10s
+  evaluation_interval: 15s
+alerting:
+  alertmanagers:
+    - static_configs:
+        - targets: []
+      scheme: http
+      timeout: 10s
+scrape_configs:
+  - job_name: prometheus
+    scrape_interval: 15s
+    scrape_timeout: 10s
+    metrics_path: /metrics
+    scheme: http
+    static_configs:
+      - targets:
+          - localhost:9090
+  - job_name: dynamic-targets
+    http_sd_configs:
+      - url: http://l0-initial-validator:9000/targets
+      - url: http://l1-initial-validator:9000/target
```

Comment:
what's the difference between `targets` and `target`?

Reply:
It should be `targets` on both, thanks for catching it, I changed it without checking if it's working.